packages/extension/src/view/devtools/pages/privateAdvertising/protectedAudience/explorableExplanation/interestGroupTransformer.ts

```diff
@@ -30,7 +30,9 @@ interface SingleAd {
 export const transformInterestGroup = (site: string) => {
   let interestGroups = SYNTHETIC_INTEREST_GROUPS[site];
 
-  interestGroups = interestGroups.map((interestGroup, index) => {
+  interestGroups = interestGroups.map((_interestGroup, index) => {
+    const interestGroup = structuredClone(_interestGroup);
```

Comment:
Do we need deep clone here? What problem are we trying to solve here?

Reply:
I noticed that without deepClone the Extension would break on the second advertiser by doing deepclone it fixes that breakage.